### PR TITLE
feat(form/inputs): control PortableTextEditor instance via ref

### PR DESCRIPTION
**Pull request by skogsmaskin**
### Description

This will allow for controlling the PortableTextEditor instance from outside of the PortableTextInput, by providing
a React ref object through the input's `props.editorRef`

Also added a Playwright Component test for this.

### What to review

Is there a better way to test the ref in the Playwright Component tests? 

After a lot of back and forth, this pattern was what I landed on.

### Testing

Should be tested automatically.

### Notes for release
* PortableTextInput can now control the underlaying PortableTextEditor instance with ref passed through props.

### CODE REVIEW

**Comment by skogsmaskin** [comment]
> No changes to documentation

🤔 Is this feature borked? I have added something new to the TS-doc.

File: packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx [hunk 1]
```diff
@@ -78,8 +79,12 @@ export function PortableTextInput(props: PortableTextInputProps) {
   } = props
 
   const {onBlur} = elementProps
+  const defaultEditorRef = useRef<PortableTextEditor | null>(null)
+  const editorRef = editorRefProp || defaultEditorRef
 
-  // Make the PTE focusable from the outside
+  // This handle will allow for natively calling .focus
```
Comment (skogsmaskin) [inline packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx:85]: I updated this comment a bit here, not really part of this PR.

**Review by pedrobonamin — APPROVED** [review]
